### DISCUSSION TITLE
中英數字元混排的括號使用方法

### DISCUSSION THREAD

**Discussion by Kaitul** [sparanoid/chinese-copywriting-guidelines#174] [Q&A]
README.md 建議使用全形中文標點，但在某些中英混排 (甚至是中文佔大部分的) 的網站或應用程式中使用的大多為半形括號。

下面列出的網頁均使用了半形括號：

> https://www.google.com/intl/zh-TW/account/about/
可以讓各項 Google 服務 (例如 Chrome 和 YouTube) 為你提供更完善的使用體驗和相關協助
https://www.google.com/intl/zh-TW/account/about/privacy-tools/
你所造訪網站的擁有者會運用 Google Analytics (分析) 收集的資料
https://www.apple.com/tw/store
店內換購須出示政府核發的有效照片證件，才能享有此項優惠 (當地法律可能要求保存這份資訊)。
螢幕的對角線長度為 5.42 吋 (iPhone 13 mini)、6.06 吋 (iPhone 13、iPhone 13 Pro)、6.68 吋 (iPhone 13 Pro Max)
https://docs.microsoft.com/zh-tw/legal/termsofuse
下列使用條款 (以下稱「TOU」) 適用於　貴用戶對 Microsoft Docs 網站 (https://docs.microsoft.com/)、Tech Profile (https://techprofile.microsoft.com/) 和任何相關服務的使用。

此外「Windows 10/11 」與「Android 11/12 」的使用者介面也盡數使用了半形括號
> 標準動態範圍 (SDR)
16.0 GB (15.9 GB 可用)
將裝置從系統解除安裝 (進階)。
只在應用程式變更我的電腦時通知我 (預設值)

> SIM 卡狀態 (SIM 卡插槽 1)
指示應用程式將廣告偵錯資訊 (例如網路流量) 寫入系統記錄

而 iOS 和 macOS 則多使用了全形括號
> 自動下載其他裝置上購買的新項目（包含免費項目）。
重新安裝 App 可還原您的資料（若 App Store 仍提供此 App）。

**Comment by xfq** [comment]
关于「全角式」「半角式」「开明式」和「居中式」标点风格的介绍，可见这篇文章：https://www.thetype.com/2018/02/14211/

**Comment by Geeyun-JY3** [comment]
要说的话，Android 用户界面简中大部分用的是全宽括号（似乎当括号内内容全为西文时就会用半宽括号括住并在左右加空格），如：
> https://cs.android.com/android/platform/superproject/+/master:packages/apps/Settings/res/values-zh-rCN/strings.xml;l=3454;drc=908d98e83388168648aca0699b6886d92d8b0d24
> SIM 卡状态（SIM 卡插槽 %1$d）

你给的几个网页对应的简中页面除苹果外用的也是全宽括号：
- > https://www.google.com/intl/zh-CN/account/about/privacy-tools/
  > 您访问的网站的所有者会使用 Google Analytics（分析）收集的数据来改善其网站 

- > https://docs.microsoft.com/zh-CN/legal/termsofuse
  > 以下使用条款（以下简称“TOU”）适用于您使用 Microsoft Docs 网站 ([https://docs.microsoft.com](https://docs.microsoft.com/zh-cn/))、Tech Profile https://techprofile.microsoft.com/) 以及任何关联服务

  （这里其实并不是都用了全宽括号，括住链接的括号就还是半宽，不太清楚这是否和微软的 style guide 有关）

不过本指北的适用场景应该更多偏向网站吧，毕竟标题下提到：
> 增强网站气质

这事和各系统（厂商）在不同语言（书写系统）的 style guide 也有关系。也不太清楚各厂商的网页和应用用户界面的 style guide 是否有所不同。

**Reply by Kaitul** [reply]
感謝您提供意見，我先前沒有用過簡中介面，經對比發現簡中 (zh-rCN) 皆為全寬，繁中 (zh-rHK 和 zh-rTW) 皆為「半寬+空白」的格式，請問這個括號的使用例外是否需要在 Readme 中註明？
